languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+public static class Badge
+{
+    public static int ComputeWidthPx(string? text,
+                                     int? fontSizePx)
+    {
+
+        return (text??"").Length*(fontSizePx??0);
```

Comment:
But then the right way to implement it would be setting 12 as the default value for the parameter... that's why I prefer to keep it simple here.

Reply:
I understand, but from a story point of view it makes less sense. What if we modified the story such that one has to return an error value (`-1`) when there is no font size specified? This could introduce people to the fact that after checking for a nullable value not being null, you could then use it as if it was not a nullable value but a regular value. What do you think?

Reply:
What about throwing an exception?

Reply:
Both the exception and my example seem a bit contrived. If it is an error not providing the value, why make it nullable after all?

Reply:
What about passing the number of columns, so that the student needs to word wrap the text to make it shorter than the requested width. The parameter is nullable, if `width=null`, there's no limit, so just print everything on a single line.

We can add a mandatory int account number, so that we'll have a mix of nullable and non nullable reference and value types. What do you think?

Something like:

`string[] PrintBadge(string name, int accountNumber, int? width)`

the return type could be a string (with \n's if students do not know arrays yet)

We'll be back to the `all or nothing` problem: there's a single method to implement.

We could design the tests so that the first cases are easy enough (i.e. single line, all data provided) and then they get progressively complex...

What do you think ? @ErikSchierboom